### PR TITLE
RDISCROWD-7781: Duplicate task check. Generate checksum

### PR DESCRIPTION
**Pull request by dchhabda**
*Issue number of the reported bug or feature request: #<number>*

**Describe your changes**
A clear and concise description of the changes you have made.

**Testing performed**
Describe the testing you have performed to ensure that the bug has been addressed, or that the new feature works as planned.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

File: pybossa/repositories/task_repository.py [hunk 1]
```diff
@@ -184,6 +185,9 @@ def save(self, element, clean_project=True):
             # set task default expiration
             if element.__class__.__name__ == "Task":
                 element.expiration = get_task_expiration(element.expiration, make_timestamp())
+                checksum = generate_checksum(element)
+                print(checksum)
```
Comment (n00rsy) [inline pybossa/repositories/task_repository.py:189]: remove `print` statement

File: pybossa/task_creator_helper.py [hunk 2]
```diff
@@ -120,3 +126,105 @@ def get_gold_answers(task):
...
+    if not project:
+        return
+
+    dup_task_config = project.info.get("duplicate_task_check")
+    if not dup_task_config:
+        return
+
+    dup_fields_configured = dup_task_config.get("duplicate_fields", [])
+    if not len(dup_fields_configured):
+        # include all task.info fields
```
Comment (n00rsy) [inline pybossa/task_creator_helper.py:199]: check does not do anything, can it be removed?

**Review by n00rsy — DISMISSED** [review]
lgtm!

File: pybossa/repositories/task_repository.py (hunk 1 above)
Comment (dchhabda) [reply pybossa/repositories/task_repository.py:189]: replacing w/ log

**Review by n00rsy — DISMISSED** [review]
lgtm!

**Review by n00rsy — DISMISSED** [review]
lgtm, thank you

**Review by n00rsy — APPROVED** [review]

**Pull request merged**